tests/Fixer/Alias/NoMixedEchoPrintFixerTest.php

```diff
@@ -282,8 +285,11 @@ public function testDefaultConfig()
      */
     public function testWrongConfig($wrongConfig)
     {
-        $this->expectException('PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException');
-        $this->expectExceptionMessageRegExp('#^\[no_mixed_echo_print\] Expected array of element \"use\" with value \"echo\" or \"print\", got \"array \([\S\s]*\)\".$#');
+        $this->setExpectedExceptionRegExp(
```

Comment:
:+1:  (although `setExpectedExceptionRegExp` is deprecated in fav. for `ExpectedExceptionRegExp` IIRC)